packages/request-manager/src/events/bootstrap.ts

```diff
@@ -18,6 +22,7 @@ export const bootstrapParser = (message: string): BootstrapEvent[] => {
         const clientStatusParsed = statusParser(clientStatus);
         return {
             progress: clientStatusParsed?.groups?.progress ?? '',
+            summary: clientStatusParsed?.groups?.summary ?? '',
```

Comment:
side note: absolutely unrelated to this PR, but I would consider some kind of validation here. first parse status, then have a look if it has all the expected properties. and if not, what then? we can't log into sentry here I guess?

Reply:
Yes, maybe that is the best approach. 
But in my experience those Tor events from ControlPort can be in different shapes and my thinking in here was to ignore all the events that are not related to what I want in order to avoid undesired errors. 

But the more we know about it the more strict we could be.

Reply:
ok, feel free to resolve this. maybe you may add some todo to the code if  you feel we might want to do this in some followup later.